TYPES.adoc

```diff
@@ -0,0 +1,94 @@
+= Content Types
+
+This is meant to resolve
+link:https://issues.jenkins-ci.org/browse/WEBSITE-21[WEBSITE-21] and define
+some of the content (read: data) types to be used for the `jenkins.io` site.
+
+
+== Timeline
+
+These content types can be thought of as a timeline of related pieces of
+content. Much like a blog or twitter feed, these pieces of content can be
+strung together on a chronological timeline.
+
+
+=== Blog Post
+
+**Template**: post
+
+**Data:**
+
+* `title`
+* `tags`
+* `body`
+
+=== Event
+
+**Template**: event
+
+**Data:**
+
+* `title`
+* `date`
+* `location`
+* `description`
+
+=== Feed Aggregation
+
+This content type is still a work-in-progress/thought, see
+link:https://issues.jenkins-ci.org/browse/WEBSITE-16[WEBSSTE-16] for more.
+
+**Template**: n/a
+
+
+== Standalone
+
+These standalone pieces of content might be linked to other pieces of content
+but not as a matter of content type so much as by links within the document
+
+
+=== Home page
+
+**Template**: homepage
+
+**Data:** n/a
+
+=== Page
+
+**Template**: page
+
+**Data:**
+
+* `title`
+* `body`
+
+=== Solution Page
+
+**Template**: page
+
+**Data:**
+
+* `title`
+* `tags`
+* `plugins`
+* `body`
+
+
+
+=== Plugin page
+
+This page is still a work-in-progress/thought experiement. Currently plugins
+generally have their home page on the link:https://wiki.jenkins-ci.org[wiki]
+but I (link:https://github.com/rtyler[rtyler]) think it would be better if they
+were included in the "main" website.
```

Comment:
Needs a solution for the editing first. I'd rather not give every plugin author commit access to the site, OTOH they should be able to change their plugins' docs at will. Link to another Git repo with more freely provided commit access?